spec/orders.mediawiki

```diff
@@ -75,27 +75,27 @@ re-subscribe to the market to synchronize the order book from scratch.
 |-
 | seq      || int   || A sequence ID || superceded in <code>orderbook</code> response
 |-
-| com     || string || the order commitment
-|-
 | marketid || string || A unique market identifier ||
 |-
 | oid     || string || the order ID ||
 |-
-| otype   || string || "l" for ''limit'', "m" for ''market'', "c" for ''cancel'' || epoch_order only
-|-
 | side    || string || "b" for ''buy'', "s" for ''sell'' || epoch_order, book_order
 |-
 | qty     || int      || order size (atoms) || epoch_order, book_order
 |-
-| rate    || int    || price rate. [[comm.mediawiki/#Rate_Encoding|message-rate encoding]]. only set on limit orders
+| rate    || int    || price rate. [[comm.mediawiki/#Rate_Encoding|message-rate encoding]] || only set on limit orders
 |-
-| tif     || string || time in force. one of "i" for ''immediate'' or "s" for ''standing''. only set on limit orders
+| tif     || string || time in force. one of "i" for ''immediate'' or "s" for ''standing'' || only set on limit orders
 |-
 | time    || int    || the order's UNIX timestamp || epoch_order, book_order
 |-
+| com     || string || the order commitment || epoch_order only
+|-
+| otype   || string || "l" for ''limit'', "m" for ''market'', "c" for ''cancel'' || epoch_order only
+|-
 | epoch   || int    || the order's epoch index || epoch_order only
 |-
-| target  || string || target order ID || only set on cancel orders
+| target  || string || target order ID || only set on cancel orders (epoch_order only)
```

Comment:
I grouped the `epoch_order` only items together at the bottom.  Note that `com` is now `epoch_order` only.